e2e/test/scenarios/question-reproductions/reproductions.cy.spec.ts

```diff
@@ -0,0 +1,152 @@
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
```

Comment:
I started a new file for TS repros. We've lost some TS files when we merged repros into large files.